.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: csharp
 
+dist: trusty
```

Comment:
According to: https://docs.travis-ci.com/user/trusty-ci-environment/
You're also meant to have "sudo: required".